Makefile

```diff
@@ -15,8 +15,12 @@ deps:
 	go get -u $(BUILDFLAGS) github.com/golang/lint/golint
 	go get $(BUILDFLAGS) github.com/vbatts/git-validation
 
-test:
-	@go test $(BUILDFLAGS) -cover ./...
+vndr:
+	go get -u github.com/LK4D4/vndr
+	vndr
+
+test: vndr
```

Comment:
At the moment I don't think we have enough (read: any) integration tests for me to say we should drop the only proper integration tests we have. In the future we should drop it IMO.